### ISSUE TITLE
Support for cryptocurrencies with symbol's maxlength > 3

### ISSUE THREAD

**Issue by pabdelhay** [django-money/django-money#600]
I'm using djmoney also with crypto currencies. Some of them have their symbols with length > 3, for instance: USDC.
CurrencyField are accepting only 3 chars, for base models and also exchange models.
Would be nice to have this maxlength increased to **6** at least, to keep using default's djmoney repo.

**Comment by benjaoming** [comment]
You should be able to set `currency_field_name` on your `MoneyField` and specify a custom `CurrencyField` with `max_length` set.

I can imagine that for someone archiving very large price histories, the storage may be at stake - but the current field was already VarChar, so whether it has max length 3 or 6 is AFAIK only about validation.

Speaking of validation, I think that everything - so far - has been 3 digits long, so changing the length would cause a slight lack of input validation, although the length of the currency code isn't full and sufficient validation. So rather than having any reliance on the input, it would be better to validate whether a currency code exists or not.

tl;dr - seems fine if anyone wants to fix this.

**Comment by ayushin** [comment]
It would be great to also support this in Rate model

**Comment by ayushin** [comment]
This should be closed too?

**Comment by benjaoming** [comment]
Thanks yes, you fixed it in #614 :tada: :pray:
